package/MDAnalysis/lib/nsgrid.pyx

```diff
@@ -901,6 +901,8 @@ cdef class FastNS(object):
                                     if probe[m] > self.box.c_pbcbox.box[m][m]:
                                         check = False
                                         break
+                                    if probe[m] == self.box.c_pbcbox.box[m][m]:
+                                        probe[m] -= 0.01*self.grid.cellsize[m]
```

Comment:
What's the justification for 0.01? Does the exact value matter? Could it be made smaller?